eng/common/scripts/artifact-metadata-parsing.ps1

```diff
@@ -100,7 +101,8 @@ function IsMavenPackageVersionPublished($pkgId, $pkgVersion, $groupId) {
 
     Write-Host "VersionCheck to maven for packageId $pkgId failed with statuscode $statusCode"
     Write-Host $statusDescription
-    exit(1)
+    #exit(1)
+    return
```

Comment:
I agree we should probably remove this exit call as it isn't a good idea to have in a function but we should return false.

Reply:
I have this only for testing purpose. Will change back
There is continueOrError flag to control whether to exit or not. The function used all over the places. If we want to change return type, we'd better to have another tasks.